data/hydrogen.default.conf

```diff
@@ -15,6 +15,7 @@
 	<useTimeLine>false</useTimeLine>
         <maxBars>400</maxBars>
         <useTheRubberbandBpmChangeEvent>false</useTheRubberbandBpmChangeEvent>
+        <dataDirectory>/usr/local/share/hydrogen/data</dataDirectory>
```

Comment:
Hi! 

If i see it correctly, then this string is taken as the default data dictionary path.  If this is true, then it would be a bad idea to hardcode it, since the default config file is used also on windows and mac os x.